### ISSUE TITLE
Language selector is not accessible with the keyboard

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#601]
You cannot set the language using the keyboard. The element cannot be accessed using the keyboard. If the drop-down menu is open, a language can be selected.
